lib/sql_dust/utils/path_utils.ex

```diff
@@ -60,20 +61,15 @@ require IEx
     {excluded, aliases} = scan_excluded(sql)
```

Comment:
This was the original function used by the select now still used by the joins maybe it's possible to make it redundent and reuse the other version.